src/components/Home/Home.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
+
+
+const Home: React.FC = () => {
+
+  return (
+    <div>
+      <h1>My Dream Home</h1>
+      <h2>Journey Edition</h2>
+      <div>
+        <h2>Menu</h2>
+        <ul>
+          <li href='#'>About</li>
+          <li href='#'>Journey</li>
+          <li href='#'>Log In</li>
+        </ul>
+      </div>
+    </div>
```

Comment:
WE will remove lines 11 to 17 to the NavBar component, so we can style and make use of animations;. Good job